docs/components.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think we should just remove this page for now and and we can add pages as we need them